schemas/test/en/test_progress_value_source_blocks_chained.json

```diff
@@ -0,0 +1,116 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "language": "en",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "001",
+    "title": "Test progress value source",
+    "theme": "default",
+    "description": "A test survey for routing based on an number equals",
+    "metadata": [
+        {
+            "name": "user_id",
+            "type": "string"
+        },
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        }
+    ],
+    "questionnaire_flow": {
+        "type": "Linear",
+        "options": {
+            "summary": {
+                "collapsible": false
+            }
+        }
+    },
+    "sections": [
+        {
+            "id": "section-1",
+            "groups": [
+                {
+                    "blocks": [
+                        {
+                            "type": "Question",
+                            "id": "s1-b1",
+                            "question": {
+                                "id": "s1-b1-q1",
+                                "title": "Section 1 Question 1",
+                                "description": [
+                                    "If you answer 0, then the second question will be skipped because of a routing rule, as well as the fourth question because of a skip condition referencing the progress of question 2, as well as the 6th question because of a routing rule referencing the progress of question 4. Otherwise, questions 2, 4 and 6 will display."
```

Comment:
> Otherwise, questions 2, 4 and 6 will display.

Found this confusing as I thought only these would display. Maybe be explict and say, `so only questions 3, 5, 7 will be displayed, otherwise, all questions will display.`?

Reply:
Amended